### PR TITLE
Updated number_of_seconds.rb

### PR DESCRIPTION
**Pull request by drewesk**
- Implemented the changes requested

[#131780275]

### CODE REVIEW

File: numbers_letters_and_variables/number_of_seconds.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+days_in_a_year = 365
+age_twenty = (seconds_in_a_minute * minutes_in_a_hour * hours_in_a_day * days_in_a_year * 20)
+age_hundred = (seconds_in_a_minute * minutes_in_a_hour * hours_in_a_day * days_in_a_year * 100)
+
+puts "There are #{seconds_in_a_minute} seconds in a minute"
+puts "There are #{minutes_in_a_hour} minutes in an hour"
+puts "There are #{hours_in_a_day} hours in a day"
+puts "There are #{days_in_a_week} days in a week"
+puts "That means there are:"
+puts "  #{seconds_in_a_minute * minutes_in_a_hour} seconds in an hour,"
```
Comment (jasonnoble) [inline numbers_letters_and_variables/number_of_seconds.rb]: Could you pull out a `seconds_in_a_hour` variable above?  IMO, I like to have interpolation do as little calculation as possible...

File: numbers_letters_and_variables/number_of_seconds.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+puts "  #{seconds_in_a_minute * minutes_in_a_hour * hours_in_a_day} seconds in a day,"
```
Comment (jasonnoble) [inline numbers_letters_and_variables/number_of_seconds.rb]: `seconds_in_a_day`

File: numbers_letters_and_variables/number_of_seconds.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+puts "  #{seconds_in_a_minute * minutes_in_a_hour * hours_in_a_day * days_in_a_week} seconds in a week"
```
Comment (jasonnoble) [inline numbers_letters_and_variables/number_of_seconds.rb]: `seconds_in_a_week`

**Review by drewesk — COMMENTED** [review]
[#131780275]

File: numbers_letters_and_variables/number_of_seconds.rb [hunk 4]
```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env ruby
+
+seconds_in_a_minute = 60
+minutes_in_a_hour = 60
+hours_in_a_day = 24
+days_in_a_week = 7
+days_in_a_year = 365
+seconds_in_day = seconds_in_a_minute * minutes_in_a_hour * hours_in_a_day
```
Comment (jasonnoble) [inline numbers_letters_and_variables/number_of_seconds.rb]:
```
seconds_in_hour = seconds_in_a_minute * minutes_in_a_hour
seconds_in_day = seconds_in_hour * hours_in_a_day
seconds_in_week = seconds_in_day * days_in_a_week
seconds_in_year = seconds_in_day * days_in_a_year
age_twenty = (seconds_in_year * 20)
age_hundred = (seconds_in_year * 100)
```

**Review by jasonnoble — CHANGES_REQUESTED** [review]

**Pull request closed without merging**